client/my-sites/plugins/plugin-upload/index.jsx

```diff
@@ -37,6 +37,7 @@ import {
 	isEligibleForAutomatedTransfer,
 	getAutomatedTransferStatus,
 } from 'calypso/state/automated-transfer/selectors';
+import isAtomicSite from 'calypso/state/selectors/is-site-automated-transfer';
```

Comment:
I don't know if this is the right selector. There is an "is-site-wpcom-atomic.js" selector which sounds more similar (and descriptive) of being an Atomic site.

https://github.com/Automattic/wp-calypso/blob/fix/upload-plugins-free-atomic/client/state/selectors/is-site-wpcom-atomic.js#L1-L14